tests/baselines/reference/arrayLiteralSpread.js

```diff
@@ -41,6 +41,6 @@ function f1() {
     var b;
 }
 function f2() {
-    var a = [];
-    var b = [5];
+    var a = [].slice().slice().slice().slice().slice();
+    var b = [5].slice().slice().slice().slice().slice();

```

Comment:
The test for which that code is generated is
```typescript
var b = [...[...[...[...[...[5]]]]]];
```
Hardly seems worth optimizing. If you write that, you deserve what we generate. :-)